### ISSUE TITLE
Fix the deploy section of .travis.yml

### ISSUE THREAD

**Issue by PeterDSteinberg** [ContinuumIO/xarray_filters#11]
There are failures in the deploy section.  From PR #8 ,see [this build](https://travis-ci.org/ContinuumIO/xarray_filters/jobs/273491022)

**Comment by PeterDSteinberg** [comment]
@gbrener - This is lower priority than the #10 issue

**Comment by gbrener** [comment]
This is fixed as of #14. Python 2.7 builds are failing due to #12 , so they are allowed to fail (via the .travis.yml "allow_failures" declaration). Closing once PR #14 is merged.
